clash-lib/src/Clash/Netlist/Types.hs

```diff
@@ -507,6 +508,10 @@ data PortMap
   --
   deriving (Show)
 
+-- | @component@ declaration
+data CompDecl
+  = VHDLComp !Text [(Text,PortDirection,HWType)]
+
```

Comment:
This looks like a rebase gone wrong. There's more lines that seem to be from other PR's, could you please check your whole PR?

Reply:
Yes, sorry about that. Not sure when it crept in :(